packages/manager/tools/sao-ovh-manager-app/lib/update-pkg.js

```diff
@@ -19,7 +19,7 @@ module.exports = ({ name, description }) => ({
     'start:watch': `lerna exec --stream --parallel --scope='@ovh-ux/manager-${name}-app' --include-dependencies -- npm run dev:watch --if-present`,
   },
   dependencies: {
-    '@ovh-ux/manager-config': '^3.0.0 || ^4.0.0',
+    '@ovh-ux/manager-config': '^4.0.0 || ^5.0.0',
```

Comment:
No webpack config generated by sao ? Or not needed to patch the language list ?

Reply:
There's already a webpack configugration located into the template `folder`.
see: https://github.com/ovh/manager/tree/master/packages/manager/tools/sao-ovh-manager-app/template

From this bare minimum configuration we didn't opt-in to the usage of `ContextReplacementPlugin`.

If you think that it must be part of the default plugins list, then maybe we can think about our default shared webpack configuration where we defined plugins usage.